db/seeds.rb

```diff
@@ -6,24 +6,24 @@
 chow = Category.create(name: 'Chow', image: "fa fa-cutlery")
 shop_all = Category.create(name: 'Shop All', image: "fa fa-archive")
 
-item1 = brewing.items.create!(title: 'Mexican Organic Dark Coffee', description: 'Bold smoky aroma with a smooth body and dark cocoa finish', price: 1200, image: File.new("#{Rails.root}/app/assets/images/beans.png"))
-item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor', price: 1500, image: File.new("#{Rails.root}/app/assets/images/coffee.bag.jpg"))
+item1 = brewing.items.create!(title: 'Mexican Organic Dark Coffee', description: 'Bold smoky aroma with a smooth body and dark cocoa finish.', price: 1200, image: File.new("#{Rails.root}/app/assets/images/beans.png"))
+item2 = brewing.items.create!(title: 'Columbian Dark Coffee', description: 'Creamy and smooth-bodied with a smoky aroma and well-balanced flavor.', price: 1500, image: File.new("#{Rails.root}/app/assets/images/coffee.bag.jpg"))
 
 item3 = merchandise.items.create!(title: 'Clear Kit', description: 'Full Kit.', price: 8000, image: File.new("#{Rails.root}/app/assets/images/merch.jpg"))
-item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder', price: 2000, image: File.new("#{Rails.root}/app/assets/images/grinder.jpg"))
-item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you', price: 12000, image: File.new("#{Rails.root}/app/assets/images/espresso-machine.jpg"))
-item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you', price: 5000, image: File.new("#{Rails.root}/app/assets/images/coffee-maker.jpg"))
+item4 = merchandise.items.create!(title: 'Coffee Grinder', description: 'Classic hand crank coffee grinder.', price: 2000, image: File.new("#{Rails.root}/app/assets/images/grinder.jpg"))
+item5 = merchandise.items.create!(title: 'Espresso Machine', description: 'Bring out the indulgent European in you.', price: 12000, image: File.new("#{Rails.root}/app/assets/images/espresso-machine.jpg"))
+item6 = merchandise.items.create!(title: 'Coffee Maker', description: 'Chrome coffee maker, for the baller in you.', price: 5000, image: File.new("#{Rails.root}/app/assets/images/coffee-maker.jpg"))
```

Comment:
Useless assignment to variable - `item6`.<br>Line is too long. [198/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.